frontend/src/components/home/midsection/lower_mid.jsx

```diff
@@ -1,62 +1,84 @@
 import React, { Component } from "react";
 import "./mid.css";
 import { Container, Row, Col } from "react-bootstrap";
+import { connect } from "react-redux";
+import { events } from "../../../actions";
+import { findNextUpcomingEvent } from "../../events/utils";
+import { getSorted, getUpcomingEvents } from "../../../reducers/selectors";
+import Moment from "react-moment";
 import footerImage1 from "../../../assets/footerImage1.jpg";
 import footerImage2 from "../../../assets/Group_at_a_table_YMIM.png";
 import footerImage3 from "../../../assets/Group_pic_YMIM.png";
 import footerImage4 from "../../../assets/chicago-illinois-skyline-skyscrapers-161963.jpeg";
 
 class LowerMid extends Component {
+  componentDidMount() {
+    this.props.fetchEvents();
+  }
+
   render() {
+    const defaultContent =
+      "Check back again soon for what's is coming up next for YMIM. If you have an event that you think YMIM should be a part of please please email: Founder@theymim.org or call: 773.941.1200";
```

Comment:
Two quick thoughts:
- Strings should be placed in a separate config or strings file. This makes it easy to update, easy to localize (tho we never will), and will clean up this code.
- For performance reasons, you should not declare static variables within a render function (or any function for that matter). At our scale, this won't really affect much. But on a much bigger site, it starts adding up. Instead, declare them at a portion of the class or file which will only be instantiated once.